apps/docs/content/dotnet/how-to/customize-runtime.mdx

```diff
@@ -16,39 +16,45 @@ import Image from '/src/components/Image';
 
 import data from '@site/static/data.json';
 
-The default .NET runtime environment contains:
+
+## Customise .NET build environment
+
+The default .NET build environment contains:
 
 - <span>{data.alpineCurrentVersion}</span>
-- selected version of .NET selected when the runtime service was created.
-- <a href="/references/cli">
-    `zCLI`
-  </a>
-  , Zerops command line tool
-- ASP .NET`and`git`
+- .NET Version
+- [zCLI](/references/cli)
+- ASP .NET and Git
+
+:::note
+If you need a different OS besides Alpine, for example Ubuntu, Set the [build.os](/dotnet/how-to/build-pipeline#os) attribute to `ubuntu`.
```

Comment:
I have a slight feeling, that this sounds like we support Alpine, Ubuntu and possibly another OS too, which we don't.